### PR TITLE
Add trampoline for AAH test

### PR DESCRIPTION
**Pull request by guillett**

### CODE REVIEW

File: app/js/services/trampolineService.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+angular.module('ddsCommon').factory('TrampolineService', function($localStorage) {
```
Comment (alexsegura) [inline app/js/services/trampolineService.js]: I think we could avoid the ["smurf naming convention"](https://blog.codinghorror.com/new-programming-jargon/)  😀

A cleaner interface would be: 
- `set(name, value = null)` → accepts a hash or a name + value
- `get(name)` → returns the whole hash, or just the key if provided. When a key is retrieved, it is removed 
- `clear()` → clears everything
- `has(name)` → allows to check if a key exists, without deleting it 
- `keys()` → returns the keys, without deleting them

File: app/js/directives/droitsEligiblesList.js [hunk 2]
```diff
@@ -12,13 +12,15 @@ angular.module('ddsApp').directive('droitEligiblesList', function() {
...
 
-angular.module('ddsApp').controller('droitsEligiblesListCtrl', function($scope) {
+angular.module('ddsApp').controller('droitsEligiblesListCtrl', function($scope, TrampolineService) {
     $scope.isNumber = _.isNumber;
     $scope.isString = _.isString;
     $scope.shouldDisplayYM2Warning = function(droit) {
         return droit.isBaseRessourcesYearMoins2 && ! $scope.ressourcesYearMoins2Captured && ! _.isString(droit.montant);
     };
 
+    $scope.setTrampoline = TrampolineService.setTrampoline;
```
Comment (alexsegura) [inline app/js/directives/droitsEligiblesList.js]: Why not: 

```
$scope.trampoline = TrampolineService;
```

File: app/js/services/trampolineService.js (hunk 1 above)
Comment (alexsegura) [reply app/js/services/trampolineService.js]: Inspired by [Symfony Flash Messages](https://symfony.com/doc/current/components/http_foundation/sessions.html#flash-messages)  😎

**Comment by guillett** [comment]
Many thanks @alexsegura. A second look ? :)

**Pull request merged**